src/components/blocks/subDivisionBlock/SubDivisionBlock.tsx

```diff
@@ -27,12 +22,12 @@ const get3ColumnClass = (index: number, length: number): string => {
   return "";
 };
 
-export const SubDivisionBlock = ({ title, id, subdivisions }: TProps) => {
+export const SubDivisionBlock = ({ subdivisions }: TProps) => {
   if (!subdivisions || subdivisions.length === 0) {
     return null;
   }
   return (
-    <Section title={title} id={id}>
+    <Section id="section-subdivisions" title="Geographic sub-divisions" count={subdivisions.length}>
```

Comment:
If Subdivision block is included multiple times (which I assume it is given it's name?), wouldn't this break as you can't have two matching ids?

Reply:
This will only ever be on any given page once.

Reply:
Maybe for later - but maybe worth calling `GeographicSubdivisionBlock` or similar